apps/rollup-example/lotsOfStylesDynamic.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+
+'use strict';
+
+import * as stylex from '@stylexjs/stylex';
+
+export const lotsOfStylesDynamic = [
```

Comment:
Good to see this setup working. I think we'll need 10-20x more dynamic styles for their impact to show up enough in the bundle benchmark

Reply:
Got it, I'm adding more styles to the file.